src/components/nav-bar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import LogOut from './auth-components/logout';
+
+export class NavBar extends React.Component {
+  render() {
+    let logOutButton;
+    if (this.props.loggedIn) {
+      logOutButton = <LogOut />;
+    }
+
+    const NavBar = (
+      <div className="nav-container">
```

Comment:
This div is unnecessary. Just set the className on the `nav` to be `nav-container` or similar.